docs/UserGuide.md

```diff
@@ -340,6 +357,18 @@ Examples:
 * `C add n/John Doe p/98765432 e/jon@jon.com t/CS2020`
 * `C add n/Rajesh Kumar p/98762342 e/raj@law.com t/Friend`
 
+:information_source: The `t/TAG` parameter is optional. You can use it as many times as you want (i.e. `C add n/John
+ Doe p/98765432 e/jon@jon.com t/CS2020 t/Brother` will add 2 tags to the contact instead of just 1).
```

Comment:
Just a recommendation: I think the sample command should include 2 tags(same for edit contact) so that the user will immediately know! (Also, can you change the sample command in AddContacts & EditContacts as well?)